src/Routes.js

```diff
@@ -50,6 +51,11 @@ function Routes({ login, signup, updateProfile, applyForJob }) {
           ? <JobList applyForJob={applyForJob}/> 
           : <Redirect to={loginLocation} />}
       </Route>
+      <Route exact path="/applications">
+        {currentUser 
+          ? <JobListForUser/> 
```

Comment:
Lets change the name of this to `UserApplications` or something like that. Since it's more about the applications